src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,49 +1,107 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_SIZE = 10;
+    private static final double INCREASING_COEFFICIENT = 1.5;
+
+    private T[] elementData = (T[]) new Object[DEFAULT_SIZE];
```

Comment:
let's initialise `elementData` field in the constructor

Reply:
done